jerry-core/ecma/operations/ecma-promise-object.c

```diff
@@ -176,16 +176,12 @@ ecma_reject_promise (ecma_value_t promise, /**< promise */
   ecma_promise_set_result (obj_p, ecma_copy_value_if_not_object (reason));
   ecma_promise_object_t *promise_p = (ecma_promise_object_t *) obj_p;
 
-  /* GC can be triggered by ecma_new_values_collection so freeing the collection
-     first and creating a new one might cause a heap after use event. */
```

Comment:
Is this change needed / valid? The ASAN job on Travis CI is complaining exactly about a heap-use-after-free.

Reply:
Sadly I can't reproduce the same issue with the same build..